gradle/elide.versions.toml

```diff
@@ -592,6 +595,7 @@ plugin_proguard = { group = "com.guardsquare", name = "proguard-gradle", version
 plugin_openrewrite = { group = "org.openrewrite", name = "plugin", version.ref = "openrewrite_plugin" }
 plugin_gradle_testretry = { group = "org.gradle", name = "test-retry-gradle-plugin", version.ref = "gradleTestRetry" }
 plugin_sigstore = { group = "dev.sigstore", name = "sigstore-gradle-sign-plugin", version.ref = "sigstore" }
+plugin_redacted = { group = "dev.zacsweers.redacted", name = "redacted-compiler-plugin-gradle", version.ref = "redacted_plugin" }
```

Comment:
our plugin isn't entirely identical to zac's, or it won't be later down the road

Reply:
Are we planning changes to it soon enough though? I'm not opposed to keeping  `redakt`, we would just need to remove the `buildSrc` from `substrate` (which I don't think it's too hard at this point)

Reply:
i'm happy keeping zac's for now, but by the same logic we could just delete it from the build entirely (but keep the code). we aren't using redakt yet _at all_, and later when we _do_, it will diverge from his fork